test/Npgsql.PluginTests/NetTopologySuiteTests.cs

```diff
@@ -62,27 +62,38 @@ public void SetUp()
         }
 
         protected override NpgsqlConnection OpenConnection(string connectionString = null)
+            => OpenConnection(connectionString);
+
+        protected NpgsqlConnection OpenConnection(string connectionString = null, Ordinates handleOrdinates = Ordinates.None)
         {
             NetTopologySuiteBootstrapper.Bootstrap();
             var conn = base.OpenConnection(connectionString);
-            conn.TypeMapper.UseNetTopologySuite();
+            conn.TypeMapper.UseNetTopologySuite(handleOrdinates: handleOrdinates);
             return conn;
         }
 
         public struct TestData
         {
+            public Ordinates Ordinates;
             public IGeometry Geometry;
             public string CommandText;
         }
 
         static readonly TestData[] Tests =
         {
-            new TestData { Geometry = new Point(new Coordinate(1d, 2500d)), CommandText = "st_makepoint(1,2500)" },
+            // Two dimensional data
+            new TestData {
+                Ordinates = Ordinates.XY,
```

Comment:
Maybe just leave it unspecified for XY and interpret null as `Ordinates.XY` in `OpenConnection()`? Just to reduce test verbosity